### ISSUE TITLE
Issue: sheeps 0.3.1 crashes dedicated forge server 1.8.9 b1764 on startup

### ISSUE THREAD

**Issue by kreezxil** [zulucap/Sheeeps#3]
crash report:
https://gist.github.com/8d86dc6e17ec2d33076a

fml server latest:
https://gist.github.com/f5d9fbaecc2649176959

**Comment by zulucap** [comment]
Just posted a fix. I will keep this open for a few days. If you have any other related issues just post here. I will be gone on vacation next week so I may not be able to release any updates until I return.

**Comment by kreezxil** [comment]
Issue: sheeps 0.3.2 crashes dedicated forge server 1.8.9 build 1764 on startup

crash report:
https://gist.github.com/8e86246bbea1a1f75143

fml server latest:
https://gist.github.com/93ae09d7974d128b0b39

If you need a minecraft server so you can test both sides of your mod, I have extra vps servers on hand, I can set up a server and give you ssh access to it including ftp. Free. Because you have been very cordial and I can see you are really trying to make your mod work. If you go for this, I'd want my website credited.

**Comment by kreezxil** [comment]
btw, if it helps, version 0.2.2-hotfix2 is the only version you made that works flawlessly for the server and the client, and by that I mean not throwing errors.

**Comment by zulucap** [comment]
Thanks for the bug reports. I did manage to get a server up to do some testing and I think I have the issue fixed, at least this one specific issue. It looked like I may have done a git derp on my local machine and actually unfixed some things that I though I had fixed as I was switching between branches for future enhancements and bug fixes on the current release. I'm still fairly new to Git and getting in to how it differs from other source controls programs I've used.

The latest fix has just been posted to CurseForge. Let me know how it goes for you.
